lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -141,20 +141,21 @@ _InsertionLocation _insertionLocationForPackageImport(
     // No directive to insert relative to; insert before the first member or
     // at the beginning of the file.
     return _InsertionLocation(unit.declarations.firstOrNull?.offset ?? 0,
-        trailingNewlineCount: 2, usePackageImports: hasPackageImports);
+        trailingNewlineCount: 2, usePackageImports: hasOnlyPackageImports);
   }
 
-  for (final importDirective in imports) {
+  hasOnlyPackageImports = !imports.any((importDirective) {
```

Comment:
Very close. But these lines are not necessary This is taking something that returns a Uri, turning it into a string, then parsing it so you can ask it for its scheme. Just get the uri and check if it's null and check the scheme.

Reply:
Initially, we are writing code like [this](https://github.com/Workiva/over_react_codemod/pull/241/commits/eaa80fded45b8034145563c9975131751c3ba8b8#diff-228c7d433e4325d61c63d9f892a715c3dadd37820e9f5a0e2c38ca3677c2ac05R150) 
` final uriContent = importDirective.uriContent;`
but after merge conflicts resolve there is updating in Analyzer 5 `uriContent` is archived and we are using `importDirective.uri.stringValue` 
and if we check scheme with uri like this 

```
hasOnlyPackageImports = !imports.any((importDirective) {
    final uri = importDirective.uri;
    return uri != null && (uri.scheme != 'package' && uri.scheme != 'dart:');
  });

```
then it's giving error `error: The getter 'scheme' isn't defined for the type 'StringLiteral'. (undefined_getter at [over_react_codemod] lib/src/intl_suggestors/intl_importer.dart:158)`

Reply:
Ah, ok. My mistake. I assumed importDirective.uri was returning a Uri, since we needed to convert it to a string. It's a StringLiteral, so this code is good.